README.md

```diff
@@ -1,2 +1,2 @@
 # awesome-books
-Creating a book library with Charles 
+Creating a book library with Charles & Patricia
```

Comment:
- [ ] Please add details to your readme file so the project can be complete.

Reply:
The necessary changes have been made on the readme file